### PR TITLE
README: some improvements

### PR DESCRIPTION
**Pull request by obnoxxx**
Just some small enhancements of the README

### CODE REVIEW

**Review by anoopcs9 — APPROVED** [review]

**Review by phlogistonjohn — COMMENTED** [review]
no comma here

File: README.md [hunk 1]
```diff
@@ -8,22 +8,24 @@
...
+$ make rpms.centos9 refspec=h:a0862d6d6de
+$ make rpms.fedora vers=39 refspec=samba-4.19.6
+```
 
-As of now 4.19, 4.20 and master branches are supported. In the absence of
+As of now, versions  4.19 and  4.20 and the master branch are supported. In the absence of the 
 *refspec* argument, the master branch is built by default. The above format is
 also applicable for other `make` targets. The *vers* argument is only valid for
-Fedora related make targets and in its absence, the default version is
+Fedora related make targets and specifies the Fedora version to build for. In its absence, the default version is
```
Comment (phlogistonjohn) [inline README.md:28]: (trying that again)  No comma between "absence" and "the default"

Comment (obnoxxx) [reply README.md]: @phlogistonjohn  wrote: 

> (trying that again) No comma between "absence" and "the default"

removed.

**Comment by obnoxxx** [comment]
@phlogistonjohn  wrote:
> no comma here

While I can't see the context here, I think I found (and fixed) the place

**Review by anoopcs9 — APPROVED** [review]

**Review by phlogistonjohn — APPROVED** [review]

**Pull request merged**